pylint/checkers/base.py

```diff
@@ -2511,14 +2511,18 @@ def _check_callable_comparison(self, node):
         left_operand, right_operand = node.left, node.ops[0][1]
         # this message should be emitted only when there is comparison of bare callable
         # with non bare callable.
-        if (
-            sum(
-                1
-                for operand in (left_operand, right_operand)
-                if isinstance(utils.safe_infer(operand), bare_callables)
-            )
-            == 1
-        ):
+        number_of_bare_callables: int = 0
```

Comment:
```suggestion
        number_of_bare_callables = 0
```
Nitpick, I think this is implicit when assigning an integer.